### PR TITLE
[stable/falco] Disable Service and AuditSink when running on EKS

### PR DESCRIPTION
**Pull request by arminc**
#### Is this a new chart
No

#### What this PR does / why we need it:

AWS EKS does not support auditregistration.k8s.io but in some cases, you still might want to run the webserver. For example to use it with [falco-eks-audit-bridge](https://github.com/xebia/falco-eks-audit-bridge). This PR allows you to enable the webserver without enabling the Service and the AuditSink.

#### Special notes for your reviewer:

@bencer @nestorsalceda

### CODE REVIEW

**Comment by bencer** [comment]
Have you considered implementing a flag outside the webserver scope to define on which platform we are running instead? Rather that a flag webserver:notOnAwsEks, something like platform:aws:eks? How other charts have approached defining on which platform are running? 

cc @nestorsalceda

**Comment by arminc** [comment]
I will have a look and adjust the PR

**Comment by iverberk** [comment]
@bencer @@nestorsalceda I've updated the chart to make the platform setting reusable across the chart and future-proof for new platforms. Please review and let me know if anything needs to be done to get this merged.

**Comment by iverberk** [comment]
@bencer @nestorsalceda I added sign-off and updated the code. Could you please review and, if ok, merge?

File: stable/falco/values.yaml [hunk 1]
```diff
@@ -82,6 +82,15 @@ ebpf:
     # Set to false if not running on Google COS
     mountEtcVolume: true
 
+# Target platform for Falco. Setting a platform to true, activates
+# certain features that are specific for the platform. For example, AWS
+# EKS does not support auditregistration.k8s.io but if you still want
+# the webserver to be UP without enabling the Service and the AuditSink then
+# set platform:aws:eks to true.
+platform:
```
Comment (nestorsalceda) [inline stable/falco/values.yaml:90]: I don't want vendor / platform specific conditions

**Review by nestorsalceda — CHANGES_REQUESTED** [review]
Don't include platform / vendor specific code.

**Pull request closed without merging**